index.html

```diff
@@ -289,6 +290,81 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="form-section" id="form-section">
+        <div class="big-form-container">
+          <div class="firsttext">
+            <p class="formtext">
+              I'm always <br />
+              interested in hearing <br />
+              about new projects, <br />
+              so if you'd like to <br />
+              chat please get in touch.
+            </p>
+          </div>
+          <div class="main-form">
+            <form id="input-form" action="https://formspree.io/f/xbjqpyeb" method="POST">
+              <!-- name section of form -->
+              <ul class="forml">
+                <li class="namepart">
+                  <input
+                    type="text"
+                    maxlength="30"
+                    placeholder="Full name"
+                    name="user_name"
+                  />
+                  <!-- email section of form  -->
+                </li>
+                <li class="namepart">
+                  <input
+                    type="email"
+                    placeholder="Email address"
+                    name="user_email"
+                    id="email"
+                  />
+                  <!-- text area of form  -->
+                </li>
+                <li class="namepart">
+                  <textarea
+                    class="textside"
+                    name="message"
+                    id="textarea"
+                  ></textarea>
+                </li>
+              </ul>
+              <button class="btnn" type="button">Get in touch</button>
+            </form>
+          </div>
+          <div class="down-icons">
+            <ul class="iconz">
+              <li class="iii">
+                <a href="https://github.com/tufoinnkuo10"
+                  ><i class="fab fa-github"></i
+                ></a>
+              </li>
+              <li class="iii">
+                <a href="https://www.linkedin.com/in/tufoin-nkuo/"
+                  ><i class="fab fa-linkedin-in"></i
+                ></a>
+              </li>
+              <li class="iii">
+                <a href="https://angel.co/jobs/home"
+                  ><i class="fab fa-angellist"></i
+                ></a>
+              </li>
+              <li class="iii">
+                <a href="https://twitter.com/home?lang=en"
+                  ><i class="fab fa-twitter"></i
```

Comment:
- [ ] As per Figma design all social media need to have the same color and centered.